hooks/useConnectedWallet.ts

```
@@ -1,5 +1,5 @@
 import { usePrivy, useWallets } from "@privy-io/react-auth";
-import { useEffect } from "react";
+import { useEffect, useMemo } from "react";
```

ESLint

[error] 'useMemo' is defined but never used.

(@typescript-eslint/no-unused-vars)